### ISSUE TITLE
Base styles have outline: 0 applied to all elements

### ISSUE THREAD

**Issue by dantrewartha** [BUNN-Digital/design-system#38]
Navigating any BUNN site with the Tab button does not show an outline around the current selected element, which is not meeting WCAG recommendations. Ideally this global style of outline: 0 or outline: none should be removed.
